src/core/text/Text.js

```diff
@@ -144,8 +144,15 @@ export default class Text extends Sprite
         const maxLineWidth = measured.maxLineWidth;
         const fontProperties = measured.fontProperties;
 
-        this.canvas.width = Math.ceil((width + (style.padding * 2)) * this.resolution);
-        this.canvas.height = Math.ceil((height + (style.padding * 2)) * this.resolution);
+        const prevTextWidth = this.textWidth;
```

Comment:
I'd call these 'previousCanvasWidth' and 'previousCanvasHeight' as that is what we are actually measuring, not the width / height of the text itself